lib/helpers.js

```diff
@@ -1,5 +1,5 @@
 cl = function(something){
```

Comment:
is this function actually used anywhere? as far as I can tell it's not and we should probably just delete it since it doesn't really seem worth having anyway. @comerc: Do you feel like updating this pr to just delete the function?